### ISSUE TITLE
Feature request: Syntax highlighter for snapshots?

### ISSUE THREAD

**Issue by lkostrowski** [Raathigesh/majestic#92]
## Feature request

Hi

Do you think it's possible to make snapshots display highlighted? It could be much more readble during comparison

**Comment by Raathigesh** [comment]
The syntax is highlighted by default by passing[ --colors ](https://jestjs.io/docs/en/cli#colors) flag to Jest and then converting the output through https://www.npmjs.com/package/ansi-to-html but I have seen some instances where they are not highlighted.

Could you please let me know which version of Jest you are using and also would you be able to share a screenshot?

If you share a sample repo with highlight not working, it would be really useful as well.

**Comment by lkostrowski** [comment]
Hmmm not sure if we are on the same page, let me clarify

I am using create-react-app with typescript. It currently is `23.6.0` but it doesn't matter.
When I run default test run, it *is* colored, but not *syntax highligted*, and AFAIK this is what `--colors` stands for (red / green diffs).

Screen:

![Image 2019-03-31 at 1 35 25 PM](https://user-images.githubusercontent.com/9268745/55288600-1867f780-53ba-11e9-882a-34dec6c302bc.png)

In case of Majestic, it's not hightlighted

![Image 2019-03-31 at 1 37 48 PM](https://user-images.githubusercontent.com/9268745/55288608-3fbec480-53ba-11e9-960b-243696096bf6.png)

But `export FORCE_COLOR=true` or passing color arg does the same, so thats perfectly fine.

What I'm thinking is some smart solution to detect HTML and highlight it. 
`ansi-to-html` might be good to try writing some jest plugin, but I don't test only only html - snapshots are also great to test some mappings which output json-like format

Anyway - not a bug!

**Comment by Raathigesh** [comment]
At least we could highlight based on the `+` and `-`.

**Comment by lkostrowski** [comment]
Yes, since `colors` is default in Jest, I think it should be default in Majestic config too. Actually IMO all default configs from jest should be passed

**Comment by briwa** [comment]
Is it just me or this has been fixed at least in `master`? ![Screen Shot 2019-05-05 at 3 51 36 PM](https://user-images.githubusercontent.com/8046636/57190528-b74db980-6f4d-11e9-836c-62257e08c585.png)

Seems like the colors are all ok on my side.

**Comment by Raathigesh** [comment]
I guess sometimes Jest doesn't send colors properly, unfortunately.
